src/message/index.ts

```diff
@@ -1 +0,0 @@
-export * from './rpc.js'
```

Comment:
We still need this file for the subpath export (`import ... from '@chainsafe/libp2p-gossipsub/message'`)

Reply:
yeah this is exactly what I got when I build lodestar 👍